backend/mora/service/orgunit.py

```diff
@@ -480,16 +644,13 @@ def get_orgunit(unitid):
     r = get_one_orgunit(c, unitid, details=UnitDetails.FULL)
 
     if not r:
-        raise exceptions.HTTPException(
-            exceptions.ErrorCodes.E_ORG_UNIT_NOT_FOUND,
-            org_unit_uuid=unitid,
-        )
+        exceptions.ErrorCodes.E_ORG_UNIT_NOT_FOUND(org_unit_uuid=unitid)
 
     return flask.jsonify(r)
 
 
 @blueprint.route('/o/<uuid:orgid>/ou/')
-@util.restrictargs('at', 'start', 'limit', 'query')
+@util.restrictargs('at', 'start', 'limit', 'query', 'tree')
```

Comment:
Add documentation for `tree`

Reply:
Also, is the `tree` parameter used anywhere in the calls from the frontend code?

Reply:
> Also, is the `tree` parameter used anywhere in the calls from the frontend code?

Not yet; the parameter allows for rendering a filtered tree, but the frontend does do that right now.

Reply:
> Not yet; the parameter allows for rendering a filtered tree, but the frontend doesn't do that right now.

Do we plan to? I'm just trying to find out why this functionality was added if it serves no purpose right now.

Reply:
> > Not yet; the parameter allows for rendering a filtered tree, but the frontend doesn't do that right now.
> 
> Do we plan to? I'm just trying to find out why this functionality was added if it serves no purpose right now.

I hope so; it'll make finding units much easier for both the mapping UI, and other uses in general.

Reply:
> > > Not yet; the parameter allows for rendering a filtered tree, but the frontend doesn't do that right now.
> > 
> > 
> > Do we plan to? I'm just trying to find out why this functionality was added if it serves no purpose right now.
> 
> I hope so; it'll make finding units much easier for both the mapping UI, and other uses in general.

I'm not a terribly big fan of adding code just for the sake of adding it. I'd rather we wait, and add this when there's an actual need for it.